### PR TITLE
General tidy up of styles for logviewer

### PR DESCRIPTION
**Pull request by poornimanayar**
### Prerequisites

- [ ] I have added steps to test this contribution in the description below

If there's an existing issue for this PR then this fixes 

### Description
The logviewer uses the _umb-package_ classes and stylesheet . I have created a new stylesheet _umb-logviewer.less_ and classes specific to logviewer. Also left aligned the saved searches to make it look a bit better

Before
image

After
image

### CODE REVIEW

**Comment by emmaburstow** [comment]
Hi @poornimanayar 

Thanks for the pull request. One of us will take a look and let you know if we need anything. ;)

Em

File: src/Umbraco.Web.UI.Client/src/less/components/umb-logviewer.less [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* PACKAGE DETAILS */
```
Comment (warrenbuckley) [inline src/Umbraco.Web.UI.Client/src/less/components/umb-logviewer.less]:
```suggestion
/* LogViewer */
```

**Comment by warrenbuckley** [comment]
All good @poornimanayar 💪 apart from a tiny/copy paste error but I will fix up after merging in (as I can't seem to commit the changes back into your PR/repo)

Thanks for the tidy up & fixes

**Comment by poornimanayar** [comment]
No worries @warrenbuckley, well spotted one though! Thanks for merging. Thats my first PR to v8 Whooohoooo

**Pull request merged**